Demo/Demo/SwiftUIComponents/CardPaymentViews/CreateOrderCardPaymentView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+struct CreateOrderCardPaymentView: View {
+
+    @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
+    let selectedMerchantIntegration: MerchantIntegration
+
+    @State private var selectedIntent: Intent = .authorize
+
+    @State private var vaultCustomerID: String = ""
+    @State var shouldVaultSelected = false
+
+
+    enum Intent: String, CaseIterable, Identifiable {
+        case authorize = "AUTHORIZE"
+        case capture = "CAPTURE"
+        var id: Self { self }
+    }
+
+    public init(
+        cardPaymentViewModel: CardPaymentViewModel,
+        selectedMerchantIntegration: MerchantIntegration
+    ) {
+        self.cardPaymentViewModel = cardPaymentViewModel
+        self.selectedMerchantIntegration = selectedMerchantIntegration
+    }
+
+    var body: some View {
+        VStack(spacing: 16) {
+            HStack {
+                Text("Create an Order")
+                    .font(.system(size: 20))
+                Spacer()
+            }
+            .frame(maxWidth: .infinity)
+            .font(.headline)
+            Picker("Intent", selection: $selectedIntent) {
+                Text("AUTHORIZE").tag(Intent.authorize)
+                Text("CAPTURE").tag(Intent.capture)
+            }
+            .pickerStyle(SegmentedPickerStyle())
+            HStack {
+                Toggle("Should Vault with Purchase", isOn: $shouldVaultSelected)
+                // TODO: turn on if vault with purchase on sample server is implemented
+                    .disabled(true)
```

Comment:
We should use hidden here since we can't use this button - confusing to have a toggle that is never able to be used displayed on the screen

Reply:
I just kept it there since there was a ticket to create UI for this.
I think we plan on implementing the server side implementation in demo app 
for e2e testing for vault with purchase.